Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeInformation.ps1

```diff
@@ -132,6 +134,13 @@ function Get-ExchangeInformation {
         foreach ($key in $getFileContentInformation.Keys) {
             if ($key -like "*.exe.config") {
                 $applicationConfigFileStatus.Add($key, $getFileContentInformation[$key])
+            } elseif ($key -like "*IanaTimeZoneMappings.xml") {
+                if (($getFileContentInformation[$key]).Present) {
+                    Write-Verbose "IanaTimeZoneMappings.xml file exists"
+                    [xml]$ianaTimeZoneMappingContent = ($getFileContentInformation[$key]).Content
```

Comment:
If the file isn't a proper `xml` doc, this could be a problem and throw an exception. Let's just return the raw content.

Reply:
Done